{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/make_artifacts.py

```diff
@@ -6,13 +6,13 @@
    Use your best judgement.
 
 """
-from pathlib import Path
 import shutil
 import sys
 import time
-from typing import Union
-
 import click
+
```

Comment:
Why did you move these import statements down?

Reply:
Trying to get closer to this:
https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement